src/main/java/com/github/princesslana/eriscasper/gateway/commands/UpdatePresence.java

```diff
@@ -0,0 +1,36 @@
+package com.github.princesslana.eriscasper.gateway.commands;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.princesslana.eriscasper.data.resource.Activity;
+import com.github.princesslana.eriscasper.gateway.ImmutablePayload;
+import com.github.princesslana.eriscasper.gateway.OpCode;
+import com.github.princesslana.eriscasper.gateway.Payload;
+import io.reactivex.annotations.Nullable;
+import org.immutables.value.Value;
+
+/**
+ * @see <a href="https://discordapp.com/developers/docs/topics/gateway#update-status">
+ *     https://discordapp.com/developers/docs/topics/gateway#update-status</a>
+ */
+@Value.Immutable
+public interface UpdatePresence extends GatewayCommand {
+  @JsonProperty("since")
+  @Nullable
+  Integer getSince();
```

Comment:
Reading the docs to me seems like this can be absent, or present and null.

Reply:
Having the option for the user to have it as null is important, Optional restricts that by leaving it out. I don't think leaving it out and having it null have the same effect.